### PR TITLE
Kill compass with fire!

### PR DESCRIPTION
**Pull request by iphands**
Before:
[11:58:52] Finished 'styles' after 21.46 s

After:
[12:59:56] Finished 'styles' after 2.16 s

We switched to using postcss/autoprefixer for vendor prefixing for and cleanCSS for minification.
Thinking about using postcss and its plugins instead of sass altogether at some point.

I had to remove the @import 'bootstrap' from the cloudforms sass. I don think we need to pull that in for CFME anyway. @lphiri can you comment there? I can pull it in if needed.

@Kinlaw @mattnolting @ryelo please take a min and make sure nothing borked

### CODE REVIEW

**Comment by iphands** [comment]
```
$ bundle install
Could not locate Gemfile

The command "bundle install" failed and exited with 10 during .

Your build has been stopped.
```

No longer need this crap! Yay!!!

File: app/styles/includes/_mixins.scss [hunk 1]
```diff
@@ -62,15 +62,15 @@
...
 }
 
 // Button mixin
 @mixin button($text-color, $bg-color) {
   font: 12px bold sans-serif;
   border: none;
   line-height: 1;
-  @include box-shadow(none);
+  box-shadow: none;
   @include button-color($text-color, $bg-color);
```
Comment (Kinlaw) [inline app/styles/includes/_mixins.scss]: Why is it that @include font-size, button-color, and media-query were not removed?

**Comment by Kinlaw** [comment]
Works on my machine, just curious about the @includes.

File: app/styles/includes/_mixins.scss (hunk 1 above)
Comment (iphands) [reply app/styles/includes/_mixins.scss]: Those mix-ins were not supplied by Compass

@mattnolting added and maintains those himself:
https://github.com/RedHatInsights/insights-frontend/blob/master/app/styles/includes/_mixins.scss#L137

**Comment by Kinlaw** [comment]
@mattnolting, @ryelo any objections to me merging into master?

**Comment by ryelo** [comment]
@Kinlaw IMO: Ship it like Amazon Prime Same Day

**Comment by Kinlaw** [comment]
Merging in two hours

**Pull request merged**